central/hash/manager/deduper.go

```diff
@@ -205,7 +190,7 @@ func (d *deduperImpl) ProcessSync() {
 			delete(d.successfullyProcessed, k)
 			// If a deployment is being removed due to reconciliation, then we will need to remove the alerts too
 			if strings.HasPrefix(k, deploymentResourceKey) {
-				alertKey := buildKey(alertResourceKey, getIDFromKey(k))
+				alertKey := eventPkg.FormatKey(alertResourceKey, eventPkg.ParseIDFromKey(k))
```

Comment:
That is the only usage of `FormatKey`, so it is difficult to judge, but couldn't we maybe design it, so that `ParseIDFromKey` is called internally in `FormatKey`?

This would make it a bit more compact
```suggestion
				alertKey := eventPkg.FormatKey(alertResourceKey, k)
```

Reply:
Thanks for the comment -- it made me realize that I didn't check for other places in code where `fmt.Sprintf("%s:%s", typ, id)` was being used. I found some additional ones and changed to use the pkg function.

Reply:
nice!